tecken/apps.py

```diff
@@ -6,17 +6,55 @@
 
 import markus
 from django_redis import get_redis_connection
-import sentry_sdk
+from sentry_sdk.integrations.boto3 import Boto3Integration
 from sentry_sdk.integrations.django import DjangoIntegration
+from sentry_sdk.integrations.redis import RedisIntegration
 from sentry_sdk.integrations.logging import ignore_logger
 
 from django.conf import settings
 from django.apps import AppConfig
 
+from tecken.libdockerflow import get_release_name
+from tecken.libsentry import (
+    build_scrub_query_string,
+    Scrubber,
+    SCRUB_RULES_DEFAULT,
+    ScrubRule,
+    set_up_sentry,
+)
+
 
 logger = logging.getLogger("django")
 
 
+SCRUB_RULES_TECKEN = [
+    # HTTP request bits
+    ScrubRule(
+        key_path="request.headers",
+        keys=["Auth-Token", "Cookie", "X-Forwarded-For", "X-Real-Ip"],
+        scrub_function="scrub",
+    ),
+    ScrubRule(
+        key_path="request.data",
+        keys=["csrfmiddlewaretoken", "client_secret"],
+        scrub_function="scrub",
+    ),
+    ScrubRule(
+        key_path="request",
+        keys=["query_string"],
+        scrub_function=build_scrub_query_string(params=["code", "state"]),
+    ),
+    ScrubRule(key_path="request", keys=["cookies"], scrub_function="scrub"),
+    # "request" shows up in exceptions as a repr which in Django includes the
+    # query_string, so best to scrub it
+    ScrubRule(
+        key_path="exception.values.[].stacktrace.frames.[].vars",
```

Comment:
These strings can get long and there's nothing that ensures they're correct. I'm not really sure what to do about that. I think I'm going to leave it for now.

_[author begins to lame lack of schema and then on a lark, does a web search ...]_

Turns out there's a schema that's used to verify the Python sentry sdk:

https://github.com/getsentry/sentry-data-schemas/blob/main/relay/event.schema.json

We can use that to verify these in a test.

Reply:
I thought about this some more. I think it's a doable idea, but I'm not going to do it in this PR.